### PR TITLE
ucad: Check integer overflow

### PR DESCRIPTION
**Pull request by sarkarchandan**
**Premise**: With this change we would like to create a check for the potential integer overflow for the number of frames sent by uca camera.

### CODE REVIEW

File: ucad.c [hunk 1]
```diff
@@ -231,9 +231,11 @@ serialize_param_spec (GParamSpec *pspec, UcaNetMessageProperty *prop)
  * image itself.
  */
 static gchar *
-ucad_zmq_create_image_header (gpointer buffer, guint width, guint height, guint pixel_size, gint num_sent, gsize *length)
+ucad_zmq_create_image_header (gpointer buffer, guint width, guint height, guint pixel_size, guint64 num_sent, gsize *length)
 {
-    /* TODO: num_sent will overflow, change! */
```
Comment (tfarago) [inline ucad.c:236]: This comment can go.

File: ucad.c [hunk 2]
```diff
@@ -242,7 +244,7 @@ ucad_zmq_create_image_header (gpointer buffer, guint width, guint height, guint
     gchar *timestamp;
 
     if (builder == NULL) {
-        builder = json_builder_new_immutable ();
+        builder = json_builder_new_immutable();
```
Comment (tfarago) [inline ucad.c]: oops, formatting change and we use space everywhere else, so please change back.

**Review by tfarago — COMMENTED** [review]
Good apart from cosmetics. :+1:

File: ucad.c (hunk 2 above)
Comment (sarkarchandan) [reply ucad.c]: Done

File: ucad.c (hunk 1 above)
Comment (sarkarchandan) [reply ucad.c:236]: This comment was removed.

**Comment by tfarago** [comment]
Looks good, can you squash the last commit to have a clean history?

**Pull request merged**